Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/CriticalCmdletsUsageDetection.yaml

```diff
@@ -0,0 +1,57 @@
+id: 5170c3c4-b8c9-485c-910d-a21d965ee181
+name: VIP Mailbox manipulation
+description: |
+  'Alert if an high important Cmdlet is executed on a VIP Mailbox as those Cmdlets can be used for data exfiltration or mailbox access.'
+requiredDataConnectors:
+  - connectorId: ESI-ExchangeAdminAuditLogEvents
+    dataTypes:
+      - Event
+severity: Low
+queryFrequency: 30m
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+status: Available
+tactics:
+  - Exfiltration
+  - Persistence
+  - Collection
+relevantTechniques:
+  - T1020
+  - T1098
+  - T1114
+query: |
+  ExchangeAdminAuditLogs
+  | where UserOriented == 'Yes'
+  | where IsVIP == true
+entityMappings:
+- entityType: Mailbox
+  fieldMappings:
+    - identifier: MailboxPrimaryAddress
+      columnName: TargetObject
+    - identifier: Upn
+      columnName: TargetObject
```

Comment:
MailboxPrimaryAddress should contain the email address of the user while UPN should have the qualified Name of the user. Column mapping thus should be different for both.
In case only one value is present, please remove the second from here.

Similar approach is applicable to all the other mappings as we. Duplication of data across different Entities or different Identifiers will not add any extra value.

Reply:
Our recommendation generally is to have UPN = Primary Email Address. So UPN Value is normally equal to Email and if it's a user without email address in a specific environment (like Account Domain in an Account/Resource model) this allow us to connect user entities. In source side, TargetObject is filed by either Email Address or UPN if email is not found.

I can remove it if you impose it.

Reply:
My only concern is around having multiple identifiers with same value. In case it is required, please continue to add it. Otherwise, we should remove it.

Reply:
Changes done on last push